crates/router/src/core/mandate.rs

```diff
@@ -164,13 +165,14 @@ where
                     maybe_customer,
                     pm_id.get_required_value("payment_method_id")?,
                     mandate_reference,
+                    network_txn_id,
                 ) {
                     let connector = new_mandate_data.connector.clone();
                     logger::debug!("{:?}", new_mandate_data);
                     resp.request
                         .set_mandate_id(api_models::payments::MandateIds {
                             mandate_id: new_mandate_data.mandate_id.clone(),
-                            connector_mandate_id: new_mandate_data.connector_mandate_id.clone(),
+                            mandate_reference_id: None, //This is not used further in case needed please handle
```

Comment:
just pass the value based on match logic

Reply:
ok